src/main/java/com/example/onlinebookstore/model/Role.java

```diff
@@ -0,0 +1,36 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "roles")
```

Comment:
I do not see sql cripts for creation of this and other tables that are part of this task, It also mentions that we should insert some data into those tables, at least `ADMIN` user

Reply:
+